lib/kosmoform/lib/resources/rds.js

```diff
@@ -47,10 +48,12 @@ var DBInstance = AWSResource.define('AWS::RDS::DBInstance', {
     },
 });
 
+var ParameterType = commonObjects.KeyValuePair('Parameter', types.string, types.string);
+
 var DBParameterGroup = AWSResource.define('AWS::RDS::DBParameterGroup', {
     Description : { type: types.string, required: true },
     Family : { type: types.string, required: true },
-    Parameters : { type: types.object },
+    Parameters : { type: ParameterType },
```

Comment:
But the [docs](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-dbparametergroup.html#cfn-rds-dbparametergroup-parameters) say string key value pairs. I guess that works since nova converts everything to a string anyway. I'm thinking maybe `types.string` should allow for anything that has a `toString` function in it now.